### PR TITLE
Travis-CI: Try to fix broken mac builds

### PR DESCRIPTION
**Pull request by ubruhin**
See #63

### CODE REVIEW

**Comment by ubruhin** [comment]
mmh....does not help... :rage:

**Comment by ubruhin** [comment]
@rnestler I have now allowed the mac builds to fail ;)

File: .travis.yml [hunk 1]
```diff
@@ -26,6 +26,9 @@ matrix:
     - os: osx
       compiler: clang
       env: BUILD_DOXYGEN=false
+  allow_failures:
+    - os: osx
+  fast_finish: true
```
Comment (rnestler) [inline .travis.yml]: Whats fast_finish?

Comment (ubruhin) [inline .travis.yml]: https://blog.travis-ci.com/2013-11-27-fast-finishing-builds/

**Pull request merged**